utils/src/bitwise_operations.rs

```diff
@@ -7,6 +7,9 @@ use std::cmp::Ordering;
 pub trait BitOps<Rhs = Self> {
     /// Bitwise XOR of two BigUint inputs
     fn bitxor(input1: &Rhs, input: &Rhs) -> Rhs;
+
```

Comment:
I think "bitwise" is much more well understood and common terminology.  so I suggest calling these

`bitwise_and` and `bitwise_xor`